Docker_files/ocsci_container/scripts/run-ocs-ci.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+mkdir -p /opt/cluster/logs_ocsci_container
+
+# Execute run-ci
+cd /opt/ocs-ci
+exec "$@" > /opt/cluster/logs_ocsci_container/output.txt
```

Comment:
```suggestion
exec "$@ --cluster-path /opt/cluster_path" > /opt/cluster/logs_ocsci_container/output.txt
```

This way we don't need to ask user to provide something which is static anyway. No?

Reply:
We can also avoid user typing the run-ci to the container and directly provide just params, so in this case it could look like:

```
exec "run-ci --cluster-path /opt/cluster_path $@" > /opt/cluster/logs_ocsci_container/output.txt
```

Of course then you need to delete `run-ci` from documentation example command.